src/composables/transactionList.ts

```diff
@@ -58,18 +61,17 @@ function updateAccountTransaction(address: Encoded.AccountAddress, transaction:
   }
 }
 
-const setTransactionsNextPage = (address: Encoded.AccountAddress, url: string) => {
+const setTransactionsNextPage = (address: string, url: string | null) => {
   transactions.value[address].nextPageUrl = url;
 };
 
 export function useTransactionList({ store }: IDefaultComposableOptions) {
-  const { aeActiveNetworkSettings } = useAeNetworkSettings();
   const { nodeNetworkId, getAeSdk } = useAeSdk({ store });
   const { isLoggedIn, accounts } = useAccounts({ store });
-  const {
-    fetchFromMiddlewareCamelCased,
-    getMiddleware,
-  } = useMiddleware();
+  const { getMiddleware } = useMiddleware();
+  const { onNetworkChange } = createNetworkWatcher();
```

Comment:
Creation of the network watcher should be done before defining the composable. Please see how it's used in other files.